### PR TITLE
Added tests for Jinja2.

### LINKED ISSUE #49: compilescss does not support jinja2
I wanted to add support for the `compilescss` command because currently the command does not support jinja2. There have been a lot of changes in django-compressor's compress command which added support for jinja2 since the compilescss here was written based on that one. I think we/I should refactor the command and inherit from the compress command. This way we can follow the changes there. The current command on django-compressor is very messy, I think it's best to refactor that on their end first and then inherit it here, but add some changes so instead of compressing it would compile SCSS.

It might also be good to add to the readme that setting `SASS_PROCESSOR_ENABLED` is the only way to use Jinja2 with `DEBUG` set to `False`.

### PR DESCRIPTION
**Pull request by AndreasBackx**
This is a follow-up on #58. Tests have been added for Jinja2. Following is a checklist of what is or has yet to be done before this PR can be merged.

- [x] Added basic `sass_src` test for Jinja2.
- [x] Add test for #54 
- [x] Add tests for management command that uses the Jinja2 engine. This will then fix #49.

### CODE REVIEW

**Comment by AndreasBackx** [comment]
Note my commit message on the latest commit:

```
Added tests for compilescss management command.

* Resolves #49.
* Fixes failing Python 2.7 Jinja2 tests.
* Multiple engines are now supported in the compilescss command.
* Added more info to README regarding Jinja2 compilescss support.
```

I've added support for using multiple engines in compilescss as well:
```bash
manage.py compilescss --engine django --engine jinja2
```

---

I stumbled upon `BaseCommand.requires_system_checks` that we could set to `False` so the command can still without needing to run the system checks. Maybe we could look into this? [`django-compressor` uses it as well](https://github.com/django-compressor/django-compressor/blob/develop/compressor/management/commands/compress.py#L311).

Shouldn't we add the license from django-compressor because we use parts of their management command's code? They also do it because they have some code from django-compress, [see their LICENSE file](https://github.com/django-compressor/django-compressor/blob/develop/LICENSE). While we're at it, can we rename the `LICENSE-MIT` file to `LICENSE`?

File: sass_processor/management/commands/compilescss.py [hunk 1]
```diff
@@ -49,7 +50,7 @@ def __init__(self):
         self.parser = None
         self.template_exts = getattr(settings, 'SASS_TEMPLATE_EXTS', ['.html'])
         self.sass_output_style = getattr(settings, 'SASS_OUTPUT_STYLE',
-            'nested' if settings.DEBUG else 'compressed')
+                                         'nested' if settings.DEBUG else 'compressed')
```
Comment (AndreasBackx) [inline sass_processor/management/commands/compilescss.py]: My code formatter did this, I can fix this in a follow-up commit along with the other odd ones to make them look less _odd_. Perhaps to something along these lines:

```python
self.sass_output_style = getattr(
    settings,
    'SASS_OUTPUT_STYLE',
    'nested' if settings.DEBUG else 'compressed'
)
```

File: sass_processor/management/commands/compilescss.py [hunk 2]
```diff
@@ -2,31 +2,32 @@
 from __future__ import unicode_literals
 
 import os
+
```
Comment (AndreasBackx) [inline sass_processor/management/commands/compilescss.py]: I apply [isort](https://github.com/timothycrosley/isort) to all of my files, maybe we should start doing this everywhere and add a check for it in Travis along with flake8?

File: tox.ini [hunk 3]
```diff
@@ -12,6 +12,7 @@ deps =
     pytest==3.0.7
     pytest-django==3.1.2
     django-compressor==2.1.1
+    jinja2==2.9.6
```
Comment (AndreasBackx) [inline tox.ini]: Currently it isn't very intuitive to test multiple versions locally. Could we revamp the `tox.ini` to make this easier? I wrote one for [django-safedelete](https://github.com/makinacorpus/django-safedelete/blob/master/tox.ini) as well.

File: sass_processor/management/commands/compilescss.py (hunk 2 above)
Comment (jrief) [inline sass_processor/management/commands/compilescss.py]: I'll check another project and use its isort config

**Comment by jrief** [comment]
Thank you so much!

**Pull request merged**